../../tests/invalid/SetEmpty_Invalid_1.whiley:5: unable to resolve name (no match for f({void})
	found: SetEmpty_Invalid_1:f : function(int) -> int)
    return f({})
           ^^^^^
